casanovo/casanovo.py

```
@@ -294,13 +302,24 @@ def train(
     output_root: Optional[str],
     verbosity: str,
     force_overwrite: bool,
+    load_all_states: Optional[bool],
 ) -> None:
     """Train a Casanovo model on your own data.
 
     TRAIN_PEAK_PATH must be one or more annoated MGF files, such as
     those provided by MassIVE-KB, from which to train a new Casnovo
     model.
     """
+    if model is None and load_weights_only is not None:
```

Ruff (0.15.1)

[error] Undefined name `load_weights_only`

(F821)